SUMMARY.md

```diff
@@ -107,6 +107,7 @@
     * [Accounting Examples](technical-resources/strategies/fixed-term-loan-manager/fixed-term-lm-accounting-examples.md)
   * [Open Term Loan Manager](technical-resources/strategies/open-term-loan-manager/open-term-loan-manager.md)
   * [DeFi Strategies](technical-resources/strategies/defi-strategies/defi-strategies.md)
+  * [Bitcoin Strategy](technical-resources/strategies/bitcoin-strategy.md)
```

Comment:
I would structure the folder structure differently and put all syrupBTC items in one section and don't mix it with the core protocol.